app/gui2/src/composables/events.ts

```diff
@@ -345,7 +345,7 @@ export function usePointer(
     },
     pointerup(e: PointerEvent) {
       if (trackedPointer.value !== e.pointerId) {
-        return
+        return false
```

Comment:
Could the same be done in `pointerdown` above?

Reply:
True, I confused it with our shortcut handlers (where false means 'not handled').